src/RatesFeeder.js

```diff
@@ -40,7 +37,7 @@ const median = values => {
 };
 
 class RatesFeeder {
-    constructor(ethereumConnection, tickers) {
+    constructor (ethereumConnection, tickers) {
```

Comment:
we should sync our prettier settings - do you have autoformat on save? I don't have these space added after fx name